pkg/my/domain.go

```diff
@@ -65,7 +67,7 @@ func Websites(client *node.Client, domainNames []string) ([]*models.Websites, er
 	}
 
 	for i := 0; i < len(domainNames); i++ { //nolint:varnamelen
-		contractAddress := string(contractAddresses[i].CandidateValue)
+		contractAddress := string(contractAddresses[i].CandidateValue[4:])
```

Comment:
Same question

Reply:
comment added 	//we remove the first 4 bytes of the response representing the string size